src/toast/ops/noise_estimation.py

```diff
@@ -84,9 +85,10 @@ class NoiseEstim(Operator):
         help="Observation detdata key for processing mask flags",
     )
 
-    mask_flag_mask = Int(
-        defaults.det_mask_invalid, help="Bit mask for raising processing mask flags"
-    )
+    # FIXME:  This trait is unused, ok to remove?
```

Comment:
This could actually provide the solution to the processing mask vs. processing flags issue. Perhaps we need a separate flag bit for the processing mask rather than lump them with other processing flags?